src/azure-cli/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -396,6 +397,9 @@ def _get_credentials(cmd,  # pylint: disable=too-many-statements
     cli_ctx = cmd.cli_ctx
     resource_not_found, registry = None, None
     try:
+        if cli_ctx is None or registry_name is None:
```

Comment:
It should be registry_name. However we can remove this code. See comment below. We should not try to recreate login_server from the dnl hash and suffix. Just raise an exception